[2024-11-04T00:23:18.349+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_Data_Preprocessing.feature_selection_task manual__2024-11-04T00:23:06.270555+00:00 [queued]>
[2024-11-04T00:23:18.354+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_Data_Preprocessing.feature_selection_task manual__2024-11-04T00:23:06.270555+00:00 [queued]>
[2024-11-04T00:23:18.354+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-11-04T00:23:18.354+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-11-04T00:23:18.354+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-11-04T00:23:18.360+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): feature_selection_task> on 2024-11-04 00:23:06.270555+00:00
[2024-11-04T00:23:18.363+0000] {standard_task_runner.py:55} INFO - Started process 112 to run task
[2024-11-04T00:23:18.365+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'DAG_Data_Preprocessing', 'feature_selection_task', 'manual__2024-11-04T00:23:06.270555+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmpc_bb6e_y']
[2024-11-04T00:23:18.366+0000] {standard_task_runner.py:83} INFO - Job 177: Subtask feature_selection_task
[2024-11-04T00:23:18.393+0000] {task_command.py:388} INFO - Running <TaskInstance: DAG_Data_Preprocessing.feature_selection_task manual__2024-11-04T00:23:06.270555+00:00 [running]> on host b77d411e2e76
[2024-11-04T00:23:18.421+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=House_Price_Prediction Team
AIRFLOW_CTX_DAG_ID=DAG_Data_Preprocessing
AIRFLOW_CTX_TASK_ID=feature_selection_task
AIRFLOW_CTX_EXECUTION_DATE=2024-11-04T00:23:06.270555+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-11-04T00:23:06.270555+00:00
[2024-11-04T00:23:18.516+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/airflowdag.py", line 188, in feature_selection_callable
    selected_features = select_correlated_features(df_encoded, numerical_features, target, threshold=0.3)
TypeError: select_correlated_features() got an unexpected keyword argument 'threshold'
[2024-11-04T00:23:18.521+0000] {taskinstance.py:1323} INFO - Marking task as UP_FOR_RETRY. dag_id=DAG_Data_Preprocessing, task_id=feature_selection_task, execution_date=20241104T002306, start_date=20241104T002318, end_date=20241104T002318
[2024-11-04T00:23:18.526+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 177 for task feature_selection_task (select_correlated_features() got an unexpected keyword argument 'threshold'; 112)
[2024-11-04T00:23:18.545+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-11-04T00:23:18.556+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
